<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openBt">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#openBt
{
    color: rgb(0, 0, 0);
	border-color:rgb(241, 236, 237);
    background:rgb(255, 255, 255);
    border:1px solid grey;
    border-radius: 10px;
}
QPushButton#openBt:hover
{
    border-color:rgb(0, 0, 0);
    background:rgb(220, 220, 220);
}
QPushButton#openBt:pressed
{
    border-color:rgb(0, 0, 0);
    background:rgb(190, 190, 190);
}
QPushButton#openBt:disabled
{
    border-color:rgb(167, 164, 170);
    background:rgb(167, 164, 170);
    color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>采集图像</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>341</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="closeBt">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>400</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#closeBt
{
    color: rgb(0, 0, 0);
	border-color:rgb(241, 236, 237);
    background:rgb(255, 255, 255);
    border:1px solid grey;
    border-radius: 10px;
}
QPushButton#closeBt:hover
{
    border-color:rgb(0, 0, 0);
    background:rgb(220, 220, 220);
}
QPushButton#closeBt:pressed
{
    border-color:rgb(0, 0, 0);
    background:rgb(190, 190, 190);
}
QPushButton#closeBt:disabled
{
    border-color:rgb(167, 164, 170);
    background:rgb(167, 164, 170);
    color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>关闭采集</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_title">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>854</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/icon/图像识别.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="saveBt">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>400</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#saveBt
{
    color: rgb(0, 0, 0);
	border-color:rgb(241, 236, 237);
    background:rgb(255, 255, 255);
    border:1px solid grey;
    border-radius: 10px;
}
QPushButton#saveBt:hover
{
    border-color:rgb(0, 0, 0);
    background:rgb(220, 220, 220);
}
QPushButton#saveBt:pressed
{
    border-color:rgb(0, 0, 0);
    background:rgb(190, 190, 190);
}
QPushButton#saveBt:disabled
{
    border-color:rgb(167, 164, 170);
    background:rgb(167, 164, 170);
    color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>拍照</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>80</y>
      <width>20</width>
      <height>401</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="recognitionBt">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>170</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#recognitionBt
{
    color: rgb(0, 0, 0);
	border-color:rgb(241, 236, 237);
    background:rgb(255, 255, 255);
    border:1px solid grey;
    border-radius: 10px;
}
QPushButton#recognitionBt:hover
{
    border-color:rgb(0, 0, 0);
    background:rgb(220, 220, 220);
}
QPushButton#recognitionBt:pressed
{
    border-color:rgb(0, 0, 0);
    background:rgb(190, 190, 190);
}
QPushButton#recognitionBt:disabled
{
    border-color:rgb(167, 164, 170);
    background:rgb(167, 164, 170);
    color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>图像识别</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>240</y>
      <width>381</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>100</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*设置组合框样式*/
QComboBox#comboBox{
    border: 2px solid gray;
    border-radius: 10px;
    padding: 1px 2px 1px 2px;  /*针对于组合框中的文本内容*/
}
/*设置右侧展开按钮的样式和图片*/
QComboBox#comboBox::drop-down{
	subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /*画一条单竖线*/
    border-top-right-radius: 5px; /* 圆角半径同 QComboBox */
	image:url(:/icon/triangle.png);
}
QComboBox#comboBox::drop-down:pressed{	
	background-color: rgb(180, 180, 180);
	image: url(:/icon/triangle.png);
}
</string>
    </property>
    <item>
     <property name="text">
      <string>动物</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>logo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>果蔬</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>植物</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>151</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Ubuntu Mono&quot;;</string>
    </property>
    <property name="text">
     <string>请选择要识
别的图像种类</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>400</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>400</y>
      <width>161</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ewm">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_ewm
{
    color: rgb(0, 0, 0);
	border-color:rgb(241, 236, 237);
    background:rgb(255, 255, 255);
    border:1px solid grey;
    border-radius: 10px;
}
QPushButton#pushButton_ewm:hover
{
    border-color:rgb(0, 0, 0);
    background:rgb(220, 220, 220);
}
QPushButton#pushButton_ewm:pressed
{
    border-color:rgb(0, 0, 0);
    background:rgb(190, 190, 190);
}
QPushButton#pushButton_ewm:disabled
{
    border-color:rgb(167, 164, 170);
    background:rgb(167, 164, 170);
    color: rgb(255,255,255);
}</string>
    </property>
    <property name="text">
     <string>填充秘钥</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
